### PR TITLE
70855- Tonnage screen & 71101 - Find your organisation  - New link / text

### PR DESCRIPTION
**Pull request by will-hope98**
And new tonnes screen

### CODE REVIEW

File: src/EA.Weee.Web.Tests.Unit/Controllers/OrganisationRegistrationControllerTests.cs [hunk 1]
```diff
@@ -555,5 +557,95 @@ public async Task PostSearchResults_WithSelectedOrganisationId_RedirectsToJoinOr
...
+            controller.ModelState.AddModelError("error", "error");
+
+            const string searchText = "company";
+            var viewModel = new TonnageTypeViewModel()
+            {
+                SearchedText = searchText
+            };
+
+            // Act
+            var result = await controller.TonnageType(searchText);
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web.Tests.Unit/Controllers/OrganisationRegistrationControllerTests.cs:610]: @will-hope98 This doesnt actually execute the POST method as you are passing the searchText to the method it execute the GET method. Update it to pass the viewModel to controller.TonnageType

File: src/EA.Weee.Web/ViewModels/OrganisationRegistration/Type/TonnageTypeViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿namespace EA.Weee.Web.ViewModels.OrganisationRegistration.Type
+{
+    using Core.Organisations;
+    using Shared;
+    using System.ComponentModel.DataAnnotations;
+
+    public class TonnageTypeViewModel : RadioButtonStringCollectionViewModel
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/ViewModels/OrganisationRegistration/Type/TonnageTypeViewModel.cs:7]: @will-hope98 We could add unit tests to test the required attribute on the model but as this is quite a simple model no real need. 

for info an example test to check for attributes that has been done before is...

`        [Theory]
        [InlineData("ReferenceDisplay", "Reference ID")]
        [InlineData("StartDate", "Start date")]
        [InlineData("EndDate", "End date")]
        [InlineData("WasteTypeValue", "Obligation type")]
        [InlineData("ProtocolValue", "Actual or protocol")]
        [InlineData("SubmittedDate", "Submitted date")]
        [InlineData("RejectedDate", "Date rejected")]
        [InlineData("ReturnedDate", "Date returned")]
        [InlineData("ApprovedDate", "Date approved")]
        [InlineData("ReturnedReason", "Reason")]
        [InlineData("RejectedReason", "Reason")]
        public void EvidenceNoteViewModel_Properties_ShouldHaveDisplayAttributes(string property, string display)
        {
            typeof(EvidenceNoteViewModel).GetProperty(property).Should()
                .BeDecoratedWith<DisplayNameAttribute>(a => a.DisplayName.Equals(display));
        }`

File: src/EA.Weee.Web/Views/OrganisationRegistration/TonnageType.cshtml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+﻿@using EA.Weee.Web.RazorHelpers
+@using MvcNavigationHelpers
+@model EA.Weee.Web.ViewModels.OrganisationRegistration.Type.TonnageTypeViewModel
+@{
+    ViewBag.Title = "How many tonnes of Electrical or Electronic Equipment (EEE) do you produce a year?";
+}
+
+
+@section full_main_content{
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Views/OrganisationRegistration/TonnageType.cshtml:9]: @will-hope98 I forgot about this (so added). Wrapping the form in @section full_main_content fixes any spacing issues as it adds the form into the correct layout.

File: src/EA.Weee.Web.Tests.Unit/Controllers/OrganisationRegistrationControllerTests.cs (hunk 1 above)
Comment (will-hope98) [reply src/EA.Weee.Web.Tests.Unit/Controllers/OrganisationRegistrationControllerTests.cs]: Done

File: src/EA.Weee.Web/Views/OrganisationRegistration/TonnageType.cshtml (hunk 3 above)
Comment (will-hope98) [reply src/EA.Weee.Web/Views/OrganisationRegistration/TonnageType.cshtml:9]: Ah ok

**Pull request merged**